contact/index.html

```diff
@@ -35,20 +35,19 @@
                 </nav>
             </div>
         </section>
-
-        <section class="hero-subpage">
-            <div class="hero-img-box">
-                <img class="hero-img" src="../assets/img/heros/hero-contact.jpg" alt="Travelers">
-            </div>
-            <div class="hero-content">
-                <span class="hero-subpage-text">
-                    <h1 class="hero-h1">Contact</h1>
-                    <h2 class="hero-h2">Contact</h2>
-
-                </span>
-            </div>
-        </section>
-
+        <div class="hero-subpage">
+            <section class="hero">
+                <div class="hero-img-box">
+                    <img class="hero-img" src="../assets/img/heros/hero-contact.jpg" alt="Travelers">
```

Comment:
Same w/alt tag. Even though this is a hero image, it's an image on the page and needs to reflect its content. That's an aspect of accessibility he may look at.

Reply:
updated